### ISSUE TITLE
bug: "Do you need to configure your work environment as `luv`?"

### ISSUE THREAD

**Issue by alezunino** [LazyVim/LazyVim#1975]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

v0.9.4

### Operating system/version

ArcoLinux

### Describe the bug

On a clean LazyVim install, when I open any .lua file such as ".config/nvim/lua/config/lazy.lua" I get a dialog asking "Do you need to configure your work environment as `luv`?". I suspected this could be related to the "checkThirdParty" lua_ls setting, but this is set to "false" in LazyVim (https://www.lazyvim.org/plugins/lsp). What can I do to remove this annoying error?

### Steps To Reproduce

1. clean install LazyVim
2. nvim .config/nvim/lua/config/lazy.lua

### Expected Behavior

This error did not appear last week.

### Repro

```Lua
git clone https://github.com/LazyVim/starter ~/.config/nvim
```

**Comment by dpetka2001** [comment]
Seems that due to this [commit](https://github.com/LuaLS/lua-language-server/commit/8e880ad5eee0178e32819819eef315d462f4ae65) it should be `checkThirdPart = "Disable"` now. Just add it to your lspconfig spec until it gets fixed in LazyVim core.

**Comment by dpetka2001** [comment]
@alezunino `lua_ls` got updated in mason. Try to update and if that solves your issue, feel free to close this.

**Comment by alezunino** [comment]
This has been fixed in lua_ls 3.7.2. Thanks!
